pkg/quarterdeck/db/models/users.go

```diff
@@ -1035,6 +1035,74 @@ func (u *User) fetchRoles(tx *sql.Tx) (err error) {
 	return rows.Err()
 }
 
+const (
+	getUserOrgRoleSQL = "SELECT r.name FROM organization_users ur JOIN roles r ON ur.role_id=r.id WHERE user_id=:userID AND organization_id=:orgID"
+	getNumOwnersSQL   = "SELECT COUNT(*) FROM organization_users WHERE organization_id=:orgID and role_id IN (SELECT id from roles where name=:ownerRole)"
+	userRoleUpdateSQL = "UPDATE organization_users SET modified=:modified, role_id=(SELECT id FROM roles WHERE name=:role) WHERE user_id=:userID AND organization_id=:orgID"
```

Comment:
Good SQL! You could use strict equality with the SELECT since we know that only one owner role will be returned but using IN is probably a bit more future proof.

Regarding `getNumOwnersSQL` -- I guess I'll find out soon but just from reading the SQL it seems like you're going to check for _at least 2_ rather than _at least 1_ since this query does not exclude the user being modified.